tasks/buildimages.py

```diff
@@ -82,19 +87,36 @@ def generate_pr_body(
 @task(
     help={
         "file_path": "path of the Gitlab configuration YAML file",
+        "image_type": "The type of image to get the tag for. Used for CI_IMAGE_ variables",
```

Comment:
What can this argument be set to ?

Reply:
As you have understood, there is no `choices` predefined, but it's maybe not that secure, I'll fix this.

Reply:
I updated this method to only accept valid image names. The list can be obtained with the `inv buildimages.update --list-image` task